src/main/java/scala_maven/ExtendedScalaContinuousCompileMojo.java

```diff
@@ -59,10 +59,9 @@ public File getTestAnalysisCacheFile() throws Exception {
         return FileUtils.fileOf(testAnalysisCacheFile, useCanonicalPath);
     }
 
-    // Latest 3.3.1 fails to find java home for Spark (apparently), so we use the logic in 3.2.2
```

Comment:
Btw, when I bumped this I pulled down spark to test:

```
[INFO] Spark Project Parent POM ........................... SUCCESS [03:17 min]
[INFO] Spark Project Tags ................................. SUCCESS [  0.087 s]
[INFO] Spark Project Sketch ............................... SUCCESS [  2.104 s]
[INFO] Spark Project Local DB ............................. SUCCESS [ 25.107 s]
[INFO] Spark Project Networking ........................... SUCCESS [ 13.861 s]
[INFO] Spark Project Shuffle Streaming Service ............ SUCCESS [  2.217 s]
[INFO] Spark Project Unsafe ............................... SUCCESS [  0.195 s]
[INFO] Spark Project Launcher ............................. SUCCESS [  8.586 s]
[INFO] Spark Project Core ................................. SUCCESS [02:12 min]
[INFO] Spark Project ML Local Library ..................... SUCCESS [ 20.054 s]
[INFO] Spark Project GraphX ............................... SUCCESS [  2.259 s]
[INFO] Spark Project Streaming ............................ SUCCESS [  0.130 s]
[INFO] Spark Project Catalyst ............................. SUCCESS [ 26.204 s]
[INFO] Spark Project SQL .................................. SUCCESS [ 36.625 s]
[INFO] Spark Project ML Library ........................... SUCCESS [  6.986 s]
[INFO] Spark Project Tools ................................ SUCCESS [  6.436 s]
[INFO] Spark Project Hive ................................. SUCCESS [ 20.064 s]
[INFO] Spark Project REPL ................................. SUCCESS [  2.212 s]
[INFO] Spark Project Assembly ............................. SUCCESS [  1.229 s]
[INFO] Kafka 0.10+ Token Provider for Streaming ........... SUCCESS [  1.742 s]
[INFO] Spark Integration for Kafka 0.10 ................... SUCCESS [ 25.637 s]
[INFO] Kafka 0.10+ Source for Structured Streaming ........ SUCCESS [  0.978 s]
[INFO] Spark Project Examples ............................. SUCCESS [  4.329 s]
[INFO] Spark Integration for Kafka 0.10 Assembly .......... SUCCESS [  0.179 s]
[INFO] Spark Avro ......................................... SUCCESS [  0.167 s]
[INFO] Spark Project Connect Common ....................... SUCCESS [ 34.614 s]
[INFO] Spark Project Connect Server ....................... SUCCESS [  2.148 s]
[INFO] Spark Protobuf ..................................... SUCCESS [  0.144 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  09:43 min
[INFO] Finished at: 2022-12-09T20:06:08+01:00
[INFO] ------------------------------------------------------------------------
```
So that was cool to see that it worked.